src/core/symbol/SymbolIdGenerator.ts

```diff
@@ -81,6 +82,36 @@ export class SymbolIdGenerator {
         return path;
     };
 
+    /**
+     * The namespace id based on the full alias name.
+     * @param fullName - the full namespace
+     */
+    public static namespaceId = (fullName: string): bigint => {
+        const path = SymbolIdGenerator.generateNamespacePath(fullName);
+        return path[path.length - 1];
+    };
+
+    /**
+     * Returns the bigint representation of an mosaic or namespace id.
+     *
+     * @param hexId - The hex id, like E74B99BA41F4AFEE (symbol.xym namespace) or 091F837E059AE13C (testnet mosaic id)
+     * @returns the bigint representation of the id
+     */
+    public static fromHex(hexId: string): bigint {
```

Comment:
similar to above, since we are using BigInt now, could convert from/to hex via native BigInt in node.js outside the SDK?

Reply:
The idea is to unify the string presentation of a namespace and mosaic id, bigint is great for internal but Rest uses strings.

Example:
```
"mosaics": [
      {
      "id": "E74B99BA41F4AFEE",
      "amount": "51000000000000"
      }
]
```

We can leave this to the client to parse those values, but they will need to be sure they are creating/parsing the string correctly, uppercase, no 0x, and with 0 paddings.  Apart from that, the process is fairly trivial, I thought I needed to do something more funky like reversing the bytes.

Reply:
I think both TS and Python are using buffer or long/bigint types in general. we might not need this simply because Rest returns hex?

Reply:
ok, both toHex, fromHex have been removed. Apps can handle the string representation